docs/Product-Guides/Assisted-Connect-Flow.md

```diff
@@ -1,45 +1,42 @@
 # Assisted Connect Flow
 
-Assisted Connect Flow (ACF) is our semi-automated product that expands coverage to 70+ providers. In contrast to our automated connections, the data for ACF providers is refreshed periodically. Our full list of coverage — for both API and ACF providers — can be found [here](../Development-Guides/Providers.md).
+Assisted API Connections is our semi-automated product that expands coverage to 70+ providers. In contrast to our [Automated API Connections](../Product-Guides/Automated-Connect-Flow.md), the data for Assisted API connections is refreshed periodically. Our full list of coverage — for both Automated API and Assisted API providers — can be found [here](../Development-Guides/Providers.md).
 
 Contact our team at `developers@tryfinch.com` to get started or for more questions regarding pricing, missing providers, etc.
 
 ---
 
-## How Assisted Connect works
+## How Assisted Connect Flow works
 ### For your user
-As with our automated integrations, your user will need to go through [Connect](./Product-Guides.md). However instead of providing their API credentials, they will need to follow a set of instructions, displayed on Connect, to manually connect your application to their employment system. 
+As with our automated integrations, your user will need to go through [Finch Connect](./Product-Guides.md). However instead of providing their API credentials, they will need to follow a set of instructions, displayed on Finch Connect, to manually connect your application to their employment system.
```

Comment:
```suggestion
As with our automated integrations, your user will need to go through [Finch Connect](./Product-Guides.md). However, instead of providing their API credentials, they will need to follow a set of instructions displayed on Finch Connect to manually connect your application to their employment system.
```

Reply:
Fixed